styles.css

```diff
@@ -213,23 +225,173 @@ p {
   grid-area: 5/1/6/2;
 }
 
-.attachment-number{
+.attachment-number {
   margin-top: 0;
   font-size: 2em;
   color: #359DDB
 }
-/* .card-section {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  gap: 10px
-}
-text-align:center;
-padding:20px; */
+
 /* SMALL TABLET STYLING */
-@media screen and (min-width: 620px) {}
+@media screen and (min-width: 620px) {
+  .card-section {
+    padding: 20px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(4, 1fr);
+  }
+
+  .one {
+    grid-area: 1/1/2/2;
+  }
+
+  .two {
+    grid-area: 1/2/2/3;
+  }
+
+  .card-container {
+    background-color: #FFFFFF;
+    position: relative;
+    display: grid;
+    grid-template-rows: repeat(5, 1fr);
+    grid-template-columns: 1fr;
+    gap: 10px;
+    height: 30em;
+    padding: 25px;
+    box-shadow: 5px 5px #BCBCBC;
+    margin-top: 25px;
+    margin-left: 25px;
+  }
+
+  .share-icon,
+  .heart-icon,
+  .person-icon,
+  .account-icon {
+    display: inline;
+  }
+}
 
 /* LARGE TABLET AND LAPTOP STYLING  */
-@media screen and (min-width: 960px) {}
+@media screen and (min-width: 960px) {
+  .card-section {
+    padding: 20px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+  }
+
+  .one {
+    grid-area: 1/1/2/2;
+  }
+
+  .two {
+    grid-area: 1/2/2/3;
+  }
+
+  .three {
+    grid-area: 1/3/3/4;
+  }
+
+  .four {
+    grid-area: 2/1/3/2;
+  }
+
+  .five {
+    grid-area: 2/2/3/3;
+  }
+
+  .six {
+    grid-area: 2/3/3/4
+  }
+
+  .seven {
+    grid-area: 3/1/4/2
+  }
+
+  .eight {
+    grid-area: 3/2/4/3;
+  }
+
+  .card-container {
+    background-color: #FFFFFF;
+    position: relative;
+    display: grid;
+    grid-template-rows: repeat(5, 1fr);
+    grid-template-columns: 1fr;
+    gap: 10px;
+    height: 30em;
+    padding: 25px;
+    box-shadow: 5px 5px #BCBCBC;
+    margin-top: 25px;
+    margin-left: 25px;
+  }
+
+  .share-icon,
+  .heart-icon,
+  .person-icon,
+  .account-icon {
+    display: inline;
+  }
+}
 
 /* DESKTOP STYLING  */
-@media screen and (min-width: 1200px) {}
+@media screen and (min-width: 1200px) {
+  .card-section {
+    padding: 20px;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
+
+  .one {
```

Comment:
is this .one .two .three etc determining each of your cards positioning when you resize the screen?  I'm not sure you need this.  I can show you mine to compare. Might save you some lines of code!